docs/en/ingest-management/agent-policies.asciidoc

```diff
@@ -305,6 +305,7 @@ Note that adding custom tags is not supported for a small set of inputs:
 * `cloud-defend`
 * `fleet-server`
 * `pf-elastic-collector`, `pf-elastic-symbolizer`, and `pf-host-agent`
+* `endpoint` inputs. Instead, utilize the advanced settings(*.advanced.document_enrichment.fields) of the Elastic Defend Integration.
```

Comment:
```suggestion
* `endpoint` inputs. Instead, use the advanced settings (`*.advanced.document_enrichment.fields`) of the {elastic-defend} Integration.
```
Just adding in the variable that we use for "Elastic Defend". Also the backticks will show the setting name in italics, as is our convention.